pyproject.toml

```diff
@@ -56,7 +53,7 @@ dev = [
     # For adding copyright headers (see addheader.yml and the readme)
     "addheader >= 0.3.0",
     # Get IDAES from main branch in Github
-    #"idaes-pse @ git+https://github.com/IDAES/idaes-pse"
+    "idaes-pse @ git+https://github.com/IDAES/idaes-pse"
```

Comment:
I believe this is the source of the installation errors (and therefore CI failures) and it should be removed - this requirement is specified in `requirements-dev.txt`.

Reply:
OK, I removed that. But it seems like some of the CI jobs are still failing. I don't know why.